core/services/workflows/delegate.go

```diff
@@ -100,9 +103,67 @@ func (d *Delegate) ServicesForSpec(ctx context.Context, spec job.Job) ([]job.Ser
 
 func NewDelegate(logger logger.Logger, registry types.CapabilitiesRegistry, legacyEVMChains legacyevm.LegacyChainContainer) *Delegate {
 	// NOTE: we temporarily do registration inside NewDelegate, this will be moved out of job specs in the future
-	_ = targets.InitializeWrite(registry, legacyEVMChains, logger)
+	err := targets.InitializeWrite(registry, legacyEVMChains, logger)
+	if err != nil {
+		logger.Errorw("could not initialize writes", err)
+	}
+
+	trigger := triggers.NewMercuryTriggerService()
+	err = registry.Add(context.Background(), trigger)
+	if err != nil {
+		logger.Errorw("could not add mercury trigger to registry", err)
+	}
+
+	go mercuryEventLoop(trigger, logger)
```

Comment:
We shouldn't merge this into develop, right? Otherwise we'll run our mercury loop with logging on every node everywhere :)

Reply:
Good point! I've moved it into ServicesForSpec which should only run when there's a relevant job registered. My preference would be to merge that if possible so we don't drift away from main too much.